File: FreeCodeCamp/Javascript/chunkyMonkey.js
// https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-algorithm-scripting/chunky-monkey
// Reprak11

function chunkArrayInGroups(arr, size) {
    let arreglo = []
    let contador = 0
    for (let i = 0 ; i < Math.floor(arr.length/size);i++){
        arreglo.push(arr.slice(contador,contador+size))
        contador += size;
    }
    if (arr.length%size != 0){
        arreglo.push(arr.slice(contador))
    }
    return arreglo;
}

console.log(chunkArrayInGroups(["a", "b", "c", "d","e"], 2));

File: FreeCodeCamp/Python/chunkyMonkey.py
# https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-algorithm-scripting/chunky-monkey
# Reprak11
import math

def chunkArrayInGroups(arr, size):
    arreglo = []
    contadora = 0
    for i in range(0,math.floor(len(arr)/size)):
        arreglo.append(arr[contadora:contadora+size])
        contadora += size
    if (len(arr)%2 != 0):
        arreglo.append(arr[contadora:])
    return arreglo

if __name__ == "__main__":
    print(chunkArrayInGroups([0, 1, 2, 3, 4, 5, 6], 3))